test/service_stubs/heroku_api_stub.rb

```diff
@@ -33,21 +33,25 @@ def two_factor?
     @body = MultiJson.decode(request.body.read) rescue {}
   end
 
-  post "/auth/reset_password" do
+  post "/password-resets" do
     MultiJson.encode({
-      message: <<-eos
-Check your inbox for the next steps.
-If you don't receive an email, and it's not in your spam folder, this could mean you signed up with a different address.
-      eos
+      created_at: Time.now.utc,
```

Comment:
Use 2 spaces for indentation in a hash, relative to the first position after the preceding left parenthesis.